src/main/features/dashboard/views/macro/claimStatus/partAdmitRejectionDefendant.njk

```diff
@@ -0,0 +1,26 @@
+{% from "externalLink.njk" import externalLink %}
+{% from "internalLink.njk" import internalLink %}
+{% from "./utilityMacros.njk" import downloadResponseLink %}
+
+{% macro partAdmitRejectedByClaimantDefendantDashboard(claim) %}
+  {{ t('{{ claimantName }} rejected your admission of {{ amount }}', { amount: claim.response.amount | numeral, claimantName: claim.claimData.claimant.name }) }}
+{% endmacro %}
+
+{% macro partAdmitRejectedByClaimantDefendantClaimDetails(claim) %}
+  <div class="status-block">
+    <h2 class="heading-small">{{ t('{{ claimantName }} has rejected your admission of £{{ amount }}',
```

Comment:
Why is the £ added explicitly here but not in partAdmitRejectionClaimant.njk?

Reply:
oops I missed this! will remove